src/Integration.Vsix.UnitTests/Commands/PackageCommandManagerTests.cs

```diff
@@ -86,10 +86,11 @@ public void PackageCommandManager_Initialize()
             testSubject.Initialize(serviceProvider.GetMefService<ITeamExplorerController>(),
                 serviceProvider.GetMefService<IProjectPropertyManager>(),
                 Mock.Of<IProjectToLanguageMapper>(),
-                Mock.Of<IOutputWindowService>());
+                Mock.Of<IOutputWindowService>(),
+                Mock.Of<IBrowserService>());
 
             // Assert
-            menuService.Commands.Should().HaveCount(allCommands.Count, "Unexpected number of commands");
+            menuService.Commands.Should().HaveCountGreaterOrEqualTo(allCommands.Count, "Unexpected number of commands");
```

Comment:
Could you explain why this change was needed?

Reply:
As I moved the commandId away from PackageCommandId.cs this test was failing as it was comparing the amount of commands there with the commands registered.

Reply:
I think we need to find a way to keep this test precise, avoid other issues of defining the Id as a const field of a class, but also move away from using enums.

Reply:
@rita-sonarsource thoughts?

Reply:
I think we should stop using PackageCommandManager as a service that knows of everything, and that will solve the problem. It should have a generic "Register command" service that the other commands can call, not the other way around. I wouldn't change it now though.